<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sf::AddItemDialog</class>
 <widget class="QWidget" name="sf::AddItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HierarchyViewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1,0">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblType">
        <property name="text">
         <string>&amp;Type</string>
        </property>
        <property name="buddy">
         <cstring>cbType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbType"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblObjectName">
        <property name="text">
         <string>&amp;Name</string>
        </property>
        <property name="buddy">
         <cstring>leObjectName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leObjectName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBoxLayout">
     <property name="title">
      <string>Box Layout</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lblBoxBefore">
        <property name="text">
         <string>Before</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbBoxBefore"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFormLayout">
     <property name="title">
      <string>Form Layout</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblFormBefore">
        <property name="text">
         <string>&amp;Before</string>
        </property>
        <property name="buddy">
         <cstring>cbFormBefore</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblFormRole">
        <property name="text">
         <string>Role</string>
        </property>
        <property name="buddy">
         <cstring>cbFormRole</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbFormRole"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblFormLabelText">
        <property name="text">
         <string>&amp;Label</string>
        </property>
        <property name="buddy">
         <cstring>leFormLabelText</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leFormLabelText"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblFormLabelBuddy">
        <property name="text">
         <string>Buddy</string>
        </property>
        <property name="buddy">
         <cstring>cbFormLabelBuddy</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cbFormLabelBuddy">
        <property name="text">
         <string>Yes/No</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbFormBefore"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbGridLayout">
     <property name="title">
      <string>Grid Layout</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
